pulsar-broker/src/test/java/org/apache/pulsar/broker/namespace/NamespaceUnloadingTest.java

```diff
@@ -34,6 +38,9 @@ public class NamespaceUnloadingTest extends BrokerTestBase {
     @BeforeMethod
     @Override
     protected void setup() throws Exception {
+        conf.setTopicLevelPoliciesEnabled(true);
+        conf.setForceDeleteNamespaceAllowed(true);
+        conf.setTopicLoadTimeoutSeconds(Integer.MAX_VALUE);
```

Comment:
Is it required?

Reply:
Yes, because the problem should caused by inflight topic creation. It's a race condition. Making the load timeout infinite can ensure 99% reproduction of this issue. Otherwise, it will be passed by timeout with retry.

Absolutely, we can give the test a shorter running time, but it is still not 99%.